portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -20,6 +20,18 @@ definition {
 		Click(locator1 = "ACSettings#KEYWORD_SEND_BUTTON");
 	}
 
+	macro assertDataSource {
```

Comment:
Hi, @AfonsoHSantos. I think the name could be `assertDataSourceOption` this macro checks for different data source types